File: Editorial/1119_Remove_Vowels_from_a_String/Java/1_Approach_String.java
class Solution {
    private boolean isVowel(char c) {
        return c == 'a' || c == 'i' || c == 'e' || c == 'o' || c == 'u';
    }
    
    public String removeVowels(String s) {
        StringBuffer ans = new StringBuffer(s.length());
        
        for (int i = 0; i < s.length(); i++) {
            if (!isVowel(s.charAt(i))) {
                ans.append(s.charAt(i));
            }
        }
        
        return ans.toString();
    }
}

File: Editorial/1119_Remove_Vowels_from_a_String/CPP/1_Approach_String.cpp
class Solution {
public:
    bool isVowel(char c) {
        return c == 'a' || c == 'i' || c == 'e' || c == 'o' || c == 'u';
    }

    string removeVowels(string s) {
        string ans;

        for (char c : s) {
            if (!isVowel(c)) {
                ans += c;
            }
        }
        
        return ans;
    }
};
